ADMINS-FOR-ISTIO.md

```diff
@@ -6,21 +6,21 @@ access.
 
 | Service | Purpose | Responsible Adults
 |---------|---------|-------------------
-| [CircleCi](https://circleci.com) | Secondary CI system | [rshriram](https://github.com/rshriram), [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson)
-| [Discourse](https://discuss.istio.io) | Discussion forums | [askmeegs](https://github.com/askmeegs), [linsun](https://github.com/linsun), [oaktowner](https://github.com/oaktowner), [rshriram](https://github.com/rshriram)
-| [Docker Hub](https://hub.docker.com) | Container registry | [rshriram](https://github.com/rshriram), [costinm](https://github.com/costinm), [mandarjog](https://github.com/mandarjog), [vadimeisenbergibm](https://github.com/vadimeisenbergibm), [frankbu](https://github.com/frankbu)
-| [GitHub](https://github.com/istio) | Source control, bug tracking | [howardjohn](https://github.com/howardjohn), [rshriram](https://github.com/rshriram), [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson)
+| [CircleCi](https://circleci.com) | Secondary CI system | [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson)
+| [Discourse](https://discuss.istio.io) | Discussion forums | [askmeegs](https://github.com/askmeegs), [linsun](https://github.com/linsun), [craigbox](https://github.com/craigbox), [rshriram](https://github.com/rshriram)
```

Comment:
Do we need to remove Shriram from admins throughout this?

Reply:
Yup, Done